### ISSUE TITLE
Разработать демонстрационный интерфейс

### ISSUE THREAD

**Issue by katunilya** [sevryukov/TimetableMSTeamsIntegration#9]
Требуется разработать демонстрационный веб-интерфейс, с помощью которого мы можем:

* просмотреть "выгруженные" из сервиса "Расписание" данные о командах с машинно читаемыми id (Guid moqs)
* по нажатию кнопки создать команду MS Teams для конкретной группы в терминах "Расписания" или для всех команд сразу (исключая те, для которых команды уже есть). Одновременно происходит создание всех встреч на семестр
* для существующих команд должна быть возможность просмотра списка встреч с возможностью их отмены и переноса по кнопке

Таким образом видится 2 "главные" страницы:

1. Страница с командами

   На этой странице пользователь видит список групп "Расписания" с кнопкой создания команды для группы (неактивна если команда уже есть) и кнопкой создания команды для всех групп. 

   Страницу предлагается грузить как статический View, кнопки должны работать асинхронно и перезагружать данные на странице. (см примечание внизу)

2. Станица со встречами команды

   На этой странице пользователь видит список всех встреч по датам, кнопку создания встреч (если они уже не созданы (?)). У каждой встречи так же должна быть кнопка ее отмены и кнопка ее переноса на другое время.

---

Потенциально, для того, чтобы не захламлять API MVC компонентами, можно сделать фронт на React, тогда @hypnospinner может развернуть систему для демонстрации в докере со всеми нужными компонентами.

**Comment by sevryukov** [comment]
@hypnospinner Планируется ли создать мокап и если да, то в рамках этой Issue или отдельной?

**Comment by katunilya** [comment]
@sevryukov нет, мок будет предоставлен на уровне инфраструктуры. Мы будем "подключаться" к системе расписания через интерфейс-репозиторий, который фактически будет предоставлять некоторый мок данных. В итоге имеем рабочий абстрактный интерфейс, замещенный через DI на фактический мок.

В итоге тот же веб-клиент может остаться частью системы (например для дальнейшего мониторинга и "ручного" управления), т.к. для клиента все будет "по-настоящему"

**Comment by sevryukov** [comment]
@hypnospinner Да нет, я про макет (прототип) интерфейса web клиента.

**Comment by katunilya** [comment]
@sevryukov макеты не сложные и думаю, что их можно поместить сюда же

**Comment by sevryukov** [comment]
@hypnospinner @SixDevelop К слову о реакте и макетах, есть интересный инструмент: https://github.com/miragejs/miragejs
